fiwareskuld/phase/phase0_generateuserlist_community_resources.py

```diff
@@ -0,0 +1,138 @@
+#
+# You may obtain a copy of the License at:
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# For those usages not covered by the Apache version 2.0 License please
+# contact with opensource@tid.es
+#
+from datetime import datetime
+import os.path
+
+from fiwareskuld.expired_users import ExpiredUsers
+from fiwareskuld.utils import osclients
+from fiwareskuld.conf import settings
+from fiwareskuld.utils import log
+from fiwareskuld.utils import rotated_files
+from fiwareskuld.users_management import UserManager
+from fiwareskuld.conf import settings
+
+__author__ = 'henar'
+
+logger = log.init_logs('phase0')
+
+
+class CommunityUsers:
+    def __init__(self):
+        """Constructor. Create a keystone client"""
+        clients = osclients.OpenStackClients()
+        clients.override_endpoint(
+            'identity', clients.region, 'admin', settings.KEYSTONE_ENDPOINT)
+        self.keystoneclient = clients.get_keystoneclientv3()
+        self.protected = set()
+
+    def generate_community_users_regions(self):
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_community_users()
+
+        user_manager = UserManager()
+
+        with open('community_users_regions.txt', 'w') as users_to_delete:
+            for user in community_users:
+                users_to_delete.write(user.id + " " + user.name + " " + user.community_started_at + " ")
+                regions = user_manager.get_regions(user)
+                if regions:
+                    users_to_delete.write(user_manager.get_regions(user))
+                else:
+                    users_to_delete.write("Projects is not enabled")
+                users_to_delete.write("\n")
+
+    def generate_community_users_resources(self):
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_community_users()
+
+        self._get_community_resources(community_users, 'community_users_regions_resources.txt')
+
+    def generate_expired_community_users_resources(self):
+        users = ExpiredUsers('', '', '')
+
+        community_users = users.get_list_expired_community_users()
+
+        self._get_community_resources(community_users, 'expired_community_users_regions_resources.txt')
+
+    def _get_community_resources(self, users, file):
+        user_manager = UserManager()
+
+        with open(file, 'w') as users_to_delete:
+            for user in users:
+                users_to_delete.write(user.id + " " + user.name + " " + user.community_started_at + " ")
+                regions = user_manager.get_regions(user)
+                if regions and "PiraeusUHannoverSpain2Karlsk" in regions:
```

Comment:
this part of a string means that the user is registered in all regions and it causes errors since several endpoints are not working....